### PR TITLE
Add coverage for BZs 1392513/1399840

### PR DESCRIPTION
**Pull request by oshtaier**
Cover corresponding BZ

```
nosetests tests/foreman/ui/test_user.py -m test_positive_create_with_multiple_roles_and_login
.
----------------------------------------------------------------------
Ran 1 test in 67.315s

OK
```

### CODE REVIEW

**Comment by oshtaier** [comment]
```
nosetests tests/foreman/ui/test_user.py -m test_positive_create_with_multiple_roles_and_login
.
----------------------------------------------------------------------
Ran 1 test in 71.701s

OK
```

File: tests/foreman/ui/test_user.py [hunk 1]
```diff
@@ -258,6 +258,52 @@ def test_positive_create_with_all_roles(self):
...
+                        (strategy, value % role))
+                )
+            self.login.logout()
+            self.login.login(name, password)
+            self.assertTrue(self.login.is_logged())
+            self.my_account.navigate_to_entity()
+            self.assertIsNone(
+                self.user.wait_until_element(common_locators['haserror']))
+            user_name = self.user.wait_until_element(
+                locators['users.username']).get_attribute('value')
```
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: Nice way to check whether the page is broken :)

**Comment by abalakh** [comment]
Btw are you planning to raise the same PR for 6.3 later?

**Comment by oshtaier** [comment]
The bug is specific only to 6.2.z branch. But probably, it makes sense to have that coverage for future branches to prevent regression in that area

**Comment by abalakh** [comment]
@oshtaier exactly, +1 👍

File: tests/foreman/ui/test_user.py [hunk 2]
```diff
@@ -258,6 +258,52 @@ def test_positive_create_with_all_roles(self):
...
+        """Create User with multiple roles and re-login into application
+        using that user
+
+        @id: b1d96e56-42e0-4f4b-8f38-98fb72be5c64
+
+        @Assert: User is created successfully and can be used to login into
+        application without raising any errors. Also user can review his own
+        profile data.
+
+        @BZ: 1392513
```
Comment (sghai) [inline tests/foreman/ui/test_user.py]: Bz is fixed and verified. Are we still want to reference the bz id ?

**Comment by sghai** [comment]
ACK. pending a minor question

File: tests/foreman/ui/test_user.py (hunk 2 above)
Comment (abalakh) [inline tests/foreman/ui/test_user.py]: We don't have clear agreement, by imo it's better to reference it. Sometimes BZ tests follow really weird or invalid scenarios and without reference to specific BZ later on we may think the test is invalid and should be dropped. Also if the test will suddenly start to fail we'll have a reference to issue which should be reopened. And as we're referencing the issue in docstring it doesn't introduce any redundant code or performance degradation, so why not?
`@skip_if_bug_open` decorator should definitely be dropped after BZ is closed, but personally i can't see any serious drawback in leaving the link in docstring, just few small bonuses for us.

Comment (oshtaier) [inline tests/foreman/ui/test_user.py]: That is right + I did that intentionally as that test is directly targeted on BZ coverage from the table provided by Jake, so gonna do that all the time when work with it

**Comment by sghai** [comment]
ACK

**Pull request merged**